### PR TITLE
support empty tscells

### PR DESCRIPTION
**Pull request by bkerley**
This adds support for null values in tscells, both encoding and decoding (to work in both client and server).

I haven't made it work yet; there's probably some other bits in riak_kv that it needs to actually handle null values.

### CODE REVIEW

File: src/riak_pb_ts_codec.erl [hunk 1]
```diff
@@ -180,7 +182,8 @@ decode_cells([#tscell{binary_value    = undefined,
                       timestamp_value = undefined,
                       boolean_value   = undefined,
                       set_value       = Set,
-                      map_value       = undefined} | T], Acc) ->
+                      map_value       = undefined} | T], Acc)
+  when is_list(Set) ->
```
Comment (alexmoore) [inline src/riak_pb_ts_codec.erl]: So if the set isn't a list, and the map isn't a binary, error out?

Comment (macintux) [inline src/riak_pb_ts_codec.erl]: The set is an empty list by default; should this guard be `length(Set)>0`?

Comment (bkerley) [inline src/riak_pb_ts_codec.erl]: I'm checking `length(Set) > 0` in 5dc3e8f now; that way the last clause is very much undefined and not the `set_value` clause being undefined by chance.

**Comment by alexmoore** [comment]
+1 b4a3da9279729e23b8d9a24a8b9072895117f858

**Comment by bkerley** [comment]
@borshop merge

**Pull request merged**